source/interface/dav.js

```diff
@@ -42,12 +42,14 @@ function propsToStat(props, filename) {
     const resourceType = getSingleValue(getValueForKey("resourcetype", props));
     const mimeType = getSingleValue(getValueForKey("getcontenttype", props));
     const type = getValueForKey("collection", resourceType) ? "directory" : "file";
+    const etag = getSingleValue(getValueForKey("getetag", props));
     const stat = {
         filename: filename,
         basename: path.basename(filename),
         lastmod: lastMod,
         size: parseInt(rawSize, 10),
-        type: type
+        type: type,
+        etag: etag.replace(/"/g, "")
```

Comment:
Maybe `etag && .replace(/"/g, "") || null` ?

Reply:
would this work?  not sure if i followed the example you gave exactly.
```
        etag: (etag) ? etag.replace(/"/g, "") : null
```

Reply:
That's fine too of course